nexus/deployment/src/blueprint_builder.rs

```diff
@@ -329,6 +455,118 @@ impl<'a> BlueprintBuilder<'a> {
         Ok(Ensure::Added)
     }
 
+    pub fn sled_ensure_zone_nexus(
+        &mut self,
+        sled_id: Uuid,
+    ) -> Result<Ensure, Error> {
+        // If there's already a Nexus zone on this sled, do nothing.
```

Comment:
I don't think we want this.  This is different from NTP (where two can't really coexist on the same sled) and Crucible (same, for the same zpool).  It's very useful in dev/test to have multiple Nexus instances on one sled.